internal/blobtesting/faulty.go

```diff
@@ -25,13 +25,14 @@ const (
 type FaultyStorage struct {
 	base blob.Storage
 
-	fault.Set
+	*fault.Set
```

Comment:
lint will complain if the `FaultSet` `mu` is copied for use in `FaultyStorage`.  While the scope of the construction of the `mu` in `FaultSet` does not extend beyond the `NewFaultyStorage` call, a pointer will be used to avoid warnings.

Reply:
Q: would making `mu` a pointer => `*mu` solve the issue as well?

No change requested, mainly curiosity.